processed 4 tasks

init:
A: object(100)

task 1 'publish'. lines 9-31:
created: object(104)
written: object(103)

task 2 'run'. lines 33-33:
created: object(106), object(107)
written: object(105)

task 3 'run'. lines 35-35:
Error: Transaction Effects Status: Move Runtime Abort. Location: sui::dynamic_field in function definition 4 at offset 0, Abort Code: 2
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveAbort(MoveLocation { module: ModuleId { address: sui, name: Identifier("dynamic_field") }, function: 4, instruction: 0 }, 2), source: Some(VMError { major_status: ABORTED, sub_status: Some(2), message: None, exec_state: None, location: Module(ModuleId { address: sui, name: Identifier("dynamic_field") }), indices: [], offsets: [(FunctionDefinitionIndex(4), 0)] }) } }
